doc/man5/config.pod

```diff
@@ -329,198 +268,157 @@ string for EVP_set_default_properties().
 
-    default_properties = fips=yes
+ default_properties = fips=yes
 
-If the value is B<no>, nothing happens.
+If the value is B<no>, nothing happens. Using this name is deprecated, and
+if used must be the only name in the section.
 
 =back
 
-These two commands should not be used together, as there is no control
-over how they affect each other.
-The use of B<fips_mode> is strongly discouraged and is only present
-for backward compatibility with earlier OpenSSL FIPS modules.
+=head2 SSL Configuration
 
-=head2 SSL Configuration Module
+The name B<ssl_conf> in the initialization section names the section
+containing the list SSL/TLS configurations.
+As with the providers, each name in this section identifies a
+section with the configuration for that name. For example:
 
-This module has the name B<ssl_conf> which points to a section containing
-SSL configurations.
+ [ssl_configuration]
+ server = server_tls_config
+ client = client_tls_config
+ system_default = tls_system_default
 
-Each line in the SSL configuration section contains the name of the
-configuration and the section containing it.
+ [server_tls_config]
+ ... configuration for SSL/TLS servers ...
 
-Each configuration section consists of command value pairs for B<SSL_CONF>.
-Each pair will be passed to a B<SSL_CTX> or B<SSL> structure if it calls
-SSL_CTX_config() or SSL_config() with the appropriate configuration name.
+ [client_tls_config]
+ ... configuration for SSL/TLS clients ...
 
-Note: any characters before an initial dot in the configuration section are
-ignored so the same command can be used multiple times.
+The configuration name B<system_default> has a special meaning.  If it
+exists, it is applied whenever an B<SSL_CTX> object is created.  For example,
+to impose a system-wide minimum on protocol version:
 
-For example:
+ [tls_system_default]
+ MinProtocol = TLSv1.2
 
- ssl_conf = ssl_sect
+Each configuration section consists of name/value pairs as parsed
+by B<SSL_CONF_cmd(3)>, which will be called by SSL_CTX_config() or
+SSL_config(), appropriately.  Note that any characters before an initial
+dot in the configuration section are ignored, so that the same command can
+be used multiple times. This probably is most useful for loading different
+key types, as shown here:
 
- [ssl_sect]
+ [server_tls_config]
+ RSA.Certificate = server-rsa.pem
+ ECDSA.Certificate = server-ecdsa.pem
 
- server = server_section
+=head2 Engine Configuration
 
- [server_section]
+The name B<ssl_conf> in the initialization section names the section
+containing the list of ENGINE configurations.
+As with the providers, each name in this section identifies an engine
+with the configuration for that engine.
+engine-specific section is used to specify how to load the engineo,
+active it, and set other parameters.
 
- RSA.Certificate = server-rsa.pem
- ECDSA.Certificate = server-ecdsa.pem
- Ciphers = ALL:!RC4
+Within an engine section, the following names have meaning:
```

Comment:
s/have meaning/be defined/ ?

Reply:
No, because you can put other names in there and they are collected and available.  I used "have meaning" because I despise the overuse of the term "semantics"